.github/workflows/ci.yml

```diff
@@ -36,17 +36,26 @@ jobs:
       llvm_commit: ${{ steps.repo_info.outputs.llvm_commit }}
       pybind11_commit: ${{ steps.repo_info.outputs.pybind11_commit }}
       platform_config: ${{ steps.config.outputs.platforms }}
+      actor_lower: ${{ steps.set_actor_lower.outputs.actor_lower }}
```

Comment:
rather than the lower case of the actor (which is incorrect in the case of the cudaq repository), I believe what you actually want is to just use ${{ github.repository_owner }}.